common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4569,6 +4607,12 @@
     ],
     "sqlState" : "42K03"
   },
+  "PERSISTED_VIEW_READS_FROM_TEMPORARY_VIEW" : {
```

Comment:
We already have `INVALID_TEMP_OBJ_REFERENCE`. Persistent view can't reference any temp objects. This is a common restriction, not only in declarative pipeline.

Reply:
Thanks for catching - making this change.